pkg/util/azureclient/environments.go

```diff
@@ -46,6 +58,10 @@ var (
 		AppLensScope:             "https://microsoft.onmicrosoft.com/runtimehost",
 		AppLensTenantID:          "cab8a31a-1906-4287-a0d8-4eef66b95f6e",
 		AuthorityHost:            azidentity.AzureGovernment,
+		AzureRbacPDPEnvironment: AzureRbacPDPEnvironment{
+			Endpoint:   "https://%s.authorization.azure.us/providers/Microsoft.Authorization/checkAccess?api-version=2021-06-01-preview",
```

Comment:
are we missing the population of `%s` in these two endpoints?

Reply:
nope, this is intentional. checkaccessv2 only has regional endpoint urls; there's no single global url for us to call.  So how I currently do it is using fmt.Sprintf(url, region) to get the full regional url when creating the client like below:
```go
		pdpClient = remotepdp.NewRemotePDPClient(
			fmt.Sprintf(aroEnv.AzureRbacPDPEnvironment.Endpoint, env.Location()),
			aroEnv.AzureRbacPDPEnvironment.OAuthScope,
			fpClientCred,
```

Reply:
ok gotcha, thank you. The missing piece was how this was going to be used.